west.yml

```diff
@@ -255,7 +255,7 @@ manifest:
       groups:
         - hal
     - name: hal_ti
-      revision: bc8e7b99bb668cc51a3aa384448a48c48a33f8e2
+      revision: bc44265faf929dbb4f4519bf8b4dfc3258fde919
```

Comment:
```suggestion
      revision: pull/61/head
```

Reply:
@carlescufi Isn't using `pull/61/head` instead of SHA require changing PR after that. I cant see any similar references in `west.yaml` or somehow it will converted automatically upon approval of `hal_ti` PR?

Reply:
@bogdanovs yes, it will require changing the `pull/<num>/head` to a SHA once the module PR (hal_ti) is merged. But this is the correct way of doing it because:

1. Your SHA will change anyway when you merge the PR to `main` in hal_ti, because GitHub will rebase the commit on top of `main`
2. This is how Zephyr does this: https://docs.zephyrproject.org/latest/develop/modules.html#process-for-submitting-changes-to-existing-modules

Reply:
Thank you @carlescufi !

Done